package.json

```diff
@@ -40,7 +40,7 @@
     "merge-stream": "^1.0.0",
     "minimatch-all": "^1.0.2",
     "multipipe": "^0.3.0",
-    "plylog": "^0.3.0",
+    "plylog": "^0.4.0",
```

Comment:
agreed, I've just been working on that in isolation up to this point. I'll invite you to a review of that library after this.